src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -282,9 +286,7 @@
     }
   }
 
-  /**
-   * Create the GapicProviders based on the given id
-   */
+  /** Create the GapicProviders based on the given id */
```

Comment:
This kind of stuff can be minimized if you configure Eclipse to only update styling on code regions you modify rather than updating the entire file.